src/Form/index.tsx

```diff
@@ -121,6 +156,9 @@ export interface Props {
   initialStepId?: string;
   display?: 'inline' | 'modal';
   elementProps?: ElementProps;
+  formContextRef?: React.MutableRefObject<null | ReturnType<
```

Comment:
lmk what you think about the name
alternatives
`ref`
`formRef`
`contextRef`

Reply:
I'm a fan of contextRef, IMO it prevents the user from thinking its going to be a ref pointing to the actual form element & provides consistency with the callback functions, where we call its parameter "context" in docs. I also think we should pull the validate and render functions into this context instead of leaving them as global entities